scripts/check-api-compatibility.sh

```diff
@@ -9,6 +9,10 @@ package_name() {
     node -pe "require('$1/package.json').name"
 }
 
+package_name_and_dist_tag() {
+    node -pe "const pkg = require('$1/package.json'); pkg.name + '@' + pkg.publishConfig.tag"
```

Comment:
Would it be nice to default to just package name if `pkg.publishConfig.tag` is not available (letting npm default to `latest`)? Or should we error?

I'm assuming the error from `npm install` on encountering `package-name@` will be hard to trace down.

Reply:
> Would it be nice to default to just package name if pkg.publishConfig.tag is not available (letting npm default to latest)? [...] I'm assuming the error from npm install on encountering package-name@ will be hard to trace down.

npm appears to default to 'latest' when encountering a blank version specifier:
```
% npm init -y
% npm install aws-cdk-lib@
% cat package.json | grep aws-cdk-lib
    "aws-cdk-lib": "^2.0.0-alpha.0"
```